user-integration-guide/symfony-integration.md

```diff
@@ -0,0 +1,218 @@
+*  [Docker](https://docs.docker.com/get-docker/) for bundling our visualization tools
+
+## Step 1 - Creating a Symfony Application 
+
+Create a Symfony application by running the command `symfony new my_project_name`. Remember that you need the  [Symfony CLI](https://symfony.com/download) installed for the `symfony` command to work. I am calling this example `otel-php-symfony-basic-example`, so the command is as follows; `symfony new otel-php-symfony-basic-example` .
+
+## Step 2 - Require and Test Symfony Dependencies
+
+ To define our routes within our controller methods, let's require the Doctrine annotation library by running the command `composer require doctrine/annotations`.
+
+We can test that routes defined within Controllers work by creating a `HelloController.php` file within the `src\Controller` folder as follows:
+```
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class HelloController extends AbstractController
+{
+    /**
+     * @Route("/hello", name="hello")
+     */
+    public function index(): Response
+    { 
+        return new Response('Hello World');
+    }
+}
+```
+
+To check out the routes available in our current project run `php bin/console debug:router`. 
+
+Let's confirm that our application works by running the command `symfony server:start`.
+
+You can navigate to `localhost/hello` route to see the `Hello world` response returned from the HelloController index method above.
+![image](https://user-images.githubusercontent.com/22311928/110263970-7810a980-7fb8-11eb-8683-b5f2d8a82c4a.png)
+
+
+## Step 3 - Require the OpenTelemetry PHP Library
+
+For this step, we require the OpenTelemetry PHP Library by running the command `composer require open-telemetry/opentelemetry`. It is worthy of note that this command pulls in the last stable release for the library.
+
+## Step 4 - Bundle Zipkin and  Jaeger into the Application
+
+To visualize traces from our application, we have to bundle open source tracing tools [Zipkin](https://zipkin.io/) and [Jaeger](https://www.jaegertracing.io/) into our application using docker. 
+
+Let's add a `docker-compose.yaml` file in the root of our project with the content as follows:
+
+```
+version: '3.7'
+services:
+    zipkin:
+        image: openzipkin/zipkin-slim
+        ports:
+            - 9411:9411
+    jaeger:
+        image: jaegertracing/all-in-one
+        environment:
+            COLLECTOR_ZIPKIN_HTTP_PORT: 9412
+        ports:
+            - 9412:9412
+            - 16686:16686
+```
+
+To confirm that docker is installed and running on our system, we can run the hello world docker example using the command `docker run -it --rm hello-world`. If everything works well, run  `docker-compose up -d` to pull in Zipkin and Jaeger. This might take some time, depending on your internet connection speed.
```

Comment:
We might not need this if they follow the installation instructions in the `README.md`.  It might not hurt for clarity though!

Reply:
Yep, Leaving it for extra clarity.